examples/braket_features/Simulating_Advanced_OpenQASM_Programs_with_the_Local_Simulator.ipynb

```diff
@@ -0,0 +1,733 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    pow(p1) @ pow(p2) @ x q;    // sqrt x
</pre><p>I don't think this is really <code>sqrt x</code> :P</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/176/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>Ah right, I'll change that comment</p>